data/projects/development/zypper.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: Zypper
+repository: https://github.com/openSUSE/zypper
+twitter:
+website: https://en.opensuse.org/Portal:Zypper
```

Comment:
Note to reviewers: the page does not load without the `en` prefix, otherwise I would have excluded it.